### DISCUSSION TITLE
TestFlight build?

### DISCUSSION THREAD

**Discussion by ajkessel** [commetchat/commet#104] [Ideas]
Is this app mature enough for a TestFlight build for iOS?

**Comment by Airyzz** [comment]
Probably not at this point. We are definitely keen to get running on iOS as soon as possible. At the moment I dont have access to a mac or iOS device, so we havent done much testing on these platforms at all. 

If anyone with these devices is interested in testing it out, we would love to hear how it goes!

**Reply by ajkessel** [reply]
I'm happy to test it but don't have experience building or installing iOS software outside of the TestFlight deployment framework.

**Reply by Airyzz** [reply]
I dont have any experience with that either! Until i have access to mac device, anyone testing mac/ios builds is unfortunately on their own!
